common/app/redux/index.js

```diff
@@ -1,5 +1,5 @@
 export { default as reducer } from './reducer';
-export { default as actions } from './actions';
+export * from './actions';
```

Comment:
This changes the export from `import  { actions } from 'x'` to `import  { individualAction } from 'x';`